Gemfile.lock

```diff
@@ -379,33 +379,33 @@ GEM
       rspec-mocks (~> 3.10)
       rspec-support (~> 3.10)
     rspec-support (3.11.0)
-    rubocop (1.25.0)
+    rubocop (1.27.0)
       parallel (~> 1.10)
       parser (>= 3.1.0.0)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
       rexml
-      rubocop-ast (>= 1.15.1, < 2.0)
+      rubocop-ast (>= 1.16.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.15.1)
-      parser (>= 3.0.1.1)
-    rubocop-govuk (4.3.0)
-      rubocop (= 1.25.0)
-      rubocop-ast (= 1.15.1)
-      rubocop-rails (= 2.13.2)
+    rubocop-ast (1.17.0)
+      parser (>= 3.1.1.0)
+    rubocop-govuk (4.4.0)
```

Comment:
This version of rubocop unfortunately has a bug in it that causes it to error on our linter. I'd suggest reverting the bump/locking govuk-rubocop to 4.3.0 until that's fixed and then pipeline should pass for this.